server/scripts/createTwimlApp.js

```diff
@@ -5,9 +5,9 @@ const axios = require('axios');
 const process = require('node:process');
 const NGROK_URL = 'http://localhost:4040/api/tunnels';
 const ACCOUNT_SID = process.env.ACCOUNT_SID;
-const TWIML_APP_SID = process.env.TWIML_APP_SID;
 const AUTH_TOKEN = process.env.AUTH_TOKEN;
 const client = require('twilio')(ACCOUNT_SID, AUTH_TOKEN);
+const fs = require('fs');
```

Comment:
Instead of using temporary files, can we do something similar to the client ID generation, such that we generate a UUID into an env var and then persist that through the CI job?

Reply:
We can honestly just use the same env var as the client ID, it's just a simple `uuidgen` invocation.

Reply:
I only use 'fs' to write to append a line to the `env` file so we know the `TwimlAppSid` that we will remove at the end of the e2e tests. I don't really understand how UUID works for this case?

Reply:
My thought process goes something like this:
```
make a uuid using uuidgen
append it to the env vars, similarly to the client id
make a twiml app with that uuid as the "friendly name"
point the twiml app to the ngrok tunnel
to wrap up, we can delete any twiml app with the friendly name that matches the uuid
```
This way, there's no temporary files hanging about. But in the end it actually doesn't make that much of a difference.

Reply:
Aren't temp files get deleted at the end anyway? If yes, this uuid generation seems unnecessary. Unless I'm missing some other benefits.

Reply:
Yes temp files are getting deleted at the end anyways. The twimlSID env is not shared between runs.